AnalogButtons.h

```diff
@@ -35,7 +35,7 @@
 
 class Button {
 public:
-	uint16_t value;
+	int16_t value;
```

Comment:
reading can never be negative, this doesn't make any sense, not even within the context of #5